unittest/cost_factory.hpp

```diff
@@ -0,0 +1,205 @@
+///////////////////////////////////////////////////////////////////////////////
+// BSD 3-Clause License
+//
+// Copyright (C) 2018-2019, LAAS-CNRS
+// Copyright note valid unless otherwise stated in individual files.
+// All rights reserved.
+///////////////////////////////////////////////////////////////////////////////
+
+#include <pinocchio/fwd.hpp>
+
+#include "crocoddyl/multibody/costs/centroidal-momentum.hpp"
+#include "crocoddyl/multibody/costs/com-position.hpp"
+#include "crocoddyl/multibody/costs/contact-force.hpp"
+#include "crocoddyl/multibody/costs/control.hpp"
+// #include "crocoddyl/multibody/costs/cost-sum.hpp"
+// #include "crocoddyl/multibody/costs/frame-force.hpp"
```

Comment:
Why is not running it? Remember to enable the force (constructor) in the contact DAM, otherwise it doesn't work!